src/main/webapp/js/projects/react/components/ExpandedProject.js

```diff
@@ -1,4 +1,6 @@
+import TagList from './TagList.js';
 import {expandedProjectType} from '../prop_types.js';
+import MentorCard from './MentorCard.js';
```

Comment:
nit: can we order our imports a little? Maybe put this import up with the TagList one.

Reply:
I agree. Thanks for the keen eye on keeping things neat.